PropertySheets/Chakra.Cpp.props

```diff
@@ -4,11 +4,11 @@
   <PropertyGroup Label="Globals">
     <VCProjectVersion>15.0</VCProjectVersion>
     <Keyword>Win32Proj</Keyword>
-    <WindowsTargetPlatformVersion>10.0.17134.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0.17763.0</WindowsTargetPlatformVersion>
   </PropertyGroup>
 
   <PropertyGroup Label="Configuration">
-    <PlatformToolset>v141</PlatformToolset>
+    <PlatformToolset>$(DefaultPlatformToolset)</PlatformToolset>
```

Comment:
This allows using the default compiler from VS 2017 or VS 2019 now?

Reply:
Yes. This has the following behavior:
2017: Defaults to v141.
2019: Defaults to v142.

It's always overridable, of course.